src/main/java/de/tum/www1/orion/ui/util/CommitMessageChooser.kt

```diff
@@ -61,10 +61,11 @@ class CommitMessageChooser(val project: Project) :
                 label(translate("orion.dialog.commitmessagechooser.title"))
             }
             row {
-                commitMessageField = textField(
+                textField().component.text = settings.getSetting(OrionSettingsProvider.KEYS.COMMIT_MESSAGE)
+               /* commitMessageField = textField(
                     { settings.getSetting(OrionSettingsProvider.KEYS.COMMIT_MESSAGE) },
                     {}
-                ).component
+                ).component*/
```

Comment:
outcommented code, are there more changes needed here?

Reply:
the comment was old code and now is removed